cpp/open3d/t/pipelines/registration/Registration.cpp

```diff
@@ -67,7 +66,7 @@ static void GetRegistrationResultAndCorrespondences(
         result.inlier_rmse_ = 0.0;
         result.transformation_ =
                 core::Tensor::Eye(4, core::Float64, core::Device("CPU:0"));
-        return;
+        return result;
```

Comment:
It would be best if a function only has one `result` return. In this case, it fits nicely to the first type below.

```cpp
// OK, one return entry
int func() {
    int rc;
    if (xxx) {
        rc = aaa;
    } else {
        rc = bbb;
    }
    return rc;
}

// OK, symmetrical and easy to follow
int func() {
    if (xxx) {
        return aaa;
    } else {
        return bbb;
    }
}

// Not too good, unless the if condition is sanity check.
int func() {
    if (xxx) {
        return aaa;
    }
    return bbb;
}
```

Reply:
Done. Changed to type 2.